src/main/flight/imu.c

```diff
@@ -208,6 +208,10 @@ void imuResetAccelerationSum(void)
     accTimeSum = 0;
 }
 
+float getrMatValue(int x, int y) {
```

Comment:
I think the correct form when using camelcase is getRMatValue but wait for someone to confirm ;)